### PR TITLE
dwarf.CallFrameInfo: Support parsing LSDA pointers from FDEs.

### PR DESCRIPTION
**Pull request by ltfish**
Add support in dwarf.CallFrameInfo to retrieve LSDA pointers for FDEs.

### CODE REVIEW

**Comment by mdmillerii** [comment]
Is there a testcase?  I don't see any updates to scripts/readelf or test output files.

**Comment by ltfish** [comment]
Added a test case based on a real binary with C++ exceptions inside.

**Comment by ltfish** [comment]
Hello, is there any more input expected from my side to get this PR moving?

File: elftools/dwarf/callframe.py [hunk 1]
```diff
@@ -323,6 +333,31 @@ def _read_augmentation_data(self, entry_structs):
...
+        modifier = encoding & 0xf0
+
+        formats = self._eh_encoding_to_field(structs)
+
+        ptr = struct_parse(
+            Struct('Augmentation_Data',
+                   formats[basic_encoding]('LSDA_pointer')),
+            self.stream, stream_pos=stream_offset)['LSDA_pointer']
+
+        if modifier == 0:
```
Comment (eliben) [inline elftools/dwarf/callframe.py:350]: Please add a brief comment for what `modifier` means here

Comment (ltfish) [reply elftools/dwarf/callframe.py]: Thanks for pointing out. I changed `0` to `DW_EH_encoding_flags['DW_EH_PE_absptr']`. I also added additional doc-string to describe where the implemented logic is coming from.

**Comment by ltfish** [comment]
A gentle ping :)

**Comment by ltfish** [comment]
Thanks a lot!

**Pull request merged**